### DISCUSSION TITLE
@types package

### DISCUSSION THREAD

**Discussion by kdhaduk1207** [DefinitelyTyped/DefinitelyTyped#70511] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/stream/web.d.ts

In this file at Line # 469, by mistake extends the object during declaration, which is not valid.

In Current version code is like as below

DecompressionStream: infer T extends object;

But it should be like as below

DecompressionStream: infer T;

After updating it in my local everything is working file.

Currently I am facing below problem.

![2024-09-11 15_29_41-Window](https://github.com/user-attachments/assets/11afc6ff-a7f0-4e40-a013-f4e22cc9d976)

[a comment by an automated account is not shown]

**Reply by kdhaduk1207** [reply]
> Hi, we could not find a reference to the types you are talking about in this discussion. Please edit the title to include the name on npm inside square brackets.
> 
> E.g.
> 
> * `"[@typescript/vfs] Does not x, y"`
> * `"Missing x inside [node]"`
> * `"[express] Broken support for template types"`
> 
> By doing this, I can ping the folks who maintain the types you are referring to.

Modified

**Comment by Renegade334** [comment]
Duplicate: see https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/70432#discussioncomment-10558237
